pom.xml

```diff
@@ -153,6 +171,10 @@
             <id>repo.jenkins-ci.org</id>
             <url>https://repo.jenkins-ci.org/public/</url>
         </repository>
+        <repository>
```

Comment:
shouldn't be needed,

run:
`mvn incrementals:incrementalify`

https://github.com/jenkinsci/incrementals-tools

Reply:
but our plugin is currently not released, so we need this to download the checks dependencies

Reply:
That’s automatically added by the maven profile that incrementals adds behind the scenes

Reply:
can you fix this please?

Reply:
ok, done now.